### ISSUE TITLE
Does not support print symbol

### ISSUE THREAD

**Issue by linrz** [klauscfhq/signale#42]
If I want to print the value of the symbol type variables，signale will throw a type error.
```sh
TypeError: Cannot convert a Symbol value to a string

signale/signale.js:141  msg = args.join(' ');
```
Should we support the printing of symbol type variables?

**Comment by rjoydip** [comment]
@linrz Yes, this is really a bug I checked that. 👍

**Comment by klauscfhq** [comment]
This has been fixed on [`3d1b9c7`](https://github.com/klauscfhq/signale/commit/3d1b9c771b4c7ad258d021481bfb5c1a6f23df8d), for more info see #23. New release will be out soon : )
